chanfitness/chaneventstore.go

```diff
@@ -584,5 +584,10 @@ func (c *ChannelEventStore) recordFlapCount() error {
 
 	log.Debugf("recording flap count for: %v peers", len(updates))
 
+	// Exit early if there are no updates.
+	if len(updates) == 0 {
+		return nil
+	}
+
 	return c.cfg.WriteFlapCount(updates)
```

Comment:
other option is to hide this behind `WriteFlapCount` before the db tx is opened. That way it is handled for every future call site

Reply:
done!